frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -277,9 +276,9 @@ export default {
         } else if (
           document.getElementById('model-select').contains(document.activeElement) === false
         ) {
-          $('#search').focus();
+          document.getElementById('search').focus();
         }
-      });
+      }, 100);
```

Comment:
Have you tested smaller values, eg 50 ms? I'm a bit wary of introducing lag in the UI.

Reply:
I tried 10 initially but that did not work occasionally. I did actually try 50 too and it worked fine for me but thought 100 was safer and also is not in the way of the user since it's a non-blocking lag. We can reduce it to 50 if you want though. In general I agree with you on having as small a number as possible. With this specific component though, I think we can afford to have 100 here as it plays well with the hiding animation.